core/ReportRenderer/Pdf.php

```diff
@@ -136,6 +137,11 @@ public function setLocale($locale)
         }
         // WARNING: Did you read the warning above?
 
+        // if customer fonts import use import font
+        $fontPath = "plugins/ImageGraph/fonts/unifont.ttf";
+        if (file_exists($fontPath)) {
```

Comment:
Consider using `is_file` instead of `file_exists`, see https://github.com/matomo-org/matomo/issues/16552